package.json

```diff
@@ -77,16 +77,25 @@
   },
   "devDependencies": {
     "@babel/core": "7.22.8",
+    "@babel/eslint-parser": "7.22.15",
+    "@babel/preset-env": "7.22.7",
     "@babel/runtime": "7.22.6",
-    "babel-eslint": "10.1.0",
-    "babel-jest": "26.6.3",
-    "eslint": "7.32.0",
+    "@jest/globals": "29.7.0",
+    "@react-native/eslint-config": "0.72.2",
+    "@react-native/metro-config": "0.72.11",
+    "@tsconfig/react-native": "3.0.2",
+    "@types/jest": "29.5.5",
+    "@types/react": "18.0.24",
+    "@types/react-test-renderer": "18.0.1",
```

Comment:
What is this?

Reply:
These were also typescript-related dependencies that were added on #332 but had to be temporarily reverted by #344 until this PR.

Reply:
I understand, but I mean specifically the types for this dependency (react-test-renderer), are we using it in the project?

Reply:
It's being used, yes.

https://github.com/HathorNetwork/hathor-wallet-mobile/blob/3955cfd1ad406f48b9b6b09f1363f9b410a34fb5/__tests__/App.test.tsx#L9

Although it's never _necessary_ to have `@types` dependencies, they help with development work.